search/django/weedid/utils.py

```diff
@@ -0,0 +1,55 @@
+import os
+import re
+import json
+from django.core.files.storage import FileSystemStorage
+from weedid.models import Dataset, WeedidUser
+
+
+def store_tmp_image(image, image_dir):
+    fs = FileSystemStorage()
+    fs.save(os.path.join(image_dir, image.name), image)
```

Comment:
We might need to impose some file size limitations, both on individual images and on the upload as a whole.

Reply:
I suppose this could be left until later, but basic security should perhaps be prioritised?

Reply:
Can we put in a limit on individual image size for now? Let's ask at today's meeting what a reasonable limit should be.